uninstall.adoc

```diff
@@ -16,22 +16,16 @@ Note: When a cluster's authentication is no longer managed by identity configura
 [#removing-a-identityconfiguration-instance-by-using-commands]
 == Removing resources by using commands
 
-. If you have not already. ensure that your OpenShift Container Platform CLI is configured to run `oc` commands. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html/cli_tools/openshift-cli-oc#cli-getting-started[Getting started with the OpenShift CLI] in the OpenShift Container Platform documentation for more information about how to configure the `oc` commands. 
+. If you have not already, ensure that your OpenShift Container Platform CLI is configured to run `oc` commands. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html/cli_tools/openshift-cli-oc#cli-getting-started[Getting started with the OpenShift CLI] in the OpenShift Container Platform documentation for more information about how to configure the `oc` commands. 
 
-. Change to your project namespace by entering the following command. Replace _namespace_ with the name of your project namespace:
-+
-----
-oc project <namespace>
```

Comment:
Guess using project depends if we want to use some OCP terminology vs kube terminology.

Reply:
I wasn't concerned with the use of the term "project", more that we won't know which project we're referring to.. in this case the one where they installed the operator. I'm fine with an alternate fix if you want to keep `oc project` in the mix!